simulation/src/test/resources/report.md

```diff
@@ -1,36 +1,36 @@
+one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=59.0%	client_mean=PT10.986749661S	server_cpu=PT25M          	client_received=2217/2500	server_resps=2217	codes={200=1474, 500=743}
+ one_endpoint_dies_on_each_server[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=65.5%	client_mean=PT8.1875744S   	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1638, 500=862}
            one_endpoint_dies_on_each_server[UNLIMITED_ROUND_ROBIN].txt:	success=64.2%	client_mean=PT0.6S         	server_cpu=PT25M          	client_received=2500/2500	server_resps=2500	codes={200=1606, 500=894}
        simplest_possible_case[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.799924242S 	server_cpu=PT2H55M59S     	client_received=13200/13200	server_resps=13200	codes={200=13200}
            simplest_possible_case[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.797090909S 	server_cpu=PT2H55M21.6S   	client_received=13200/13200	server_resps=13200	codes={200=13200}
                      simplest_possible_case[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.797090909S 	server_cpu=PT2H55M21.6S   	client_received=13200/13200	server_resps=13200	codes={200=13200}
         slow_503s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.137438962S 	server_cpu=PT6M39.829999979S	client_received=3000/3000	server_resps=3077	codes={200=3000}
-            slow_503s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.700519617S 	server_cpu=PT34M2.382333314S	client_received=3000/3000	server_resps=3383	codes={200=3000}
+            slow_503s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.303372511S 	server_cpu=PT14M43.437666646S	client_received=3000/3000	server_resps=3179	codes={200=3000}
                       slow_503s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT1.430774859S 	server_cpu=PT1H9M33.661333314S	client_received=3000/3000	server_resps=3798	codes={200=3000}
-slowdown_and_error_thresholds[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=32.6%	client_mean=PT1M33.640484199S	server_cpu=PT10H34M22.666666616S	client_received=10000/10000	server_resps=10000	codes={200=3261, 500=6739}
-    slowdown_and_error_thresholds[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=1.2%	client_mean=PT24.092490733S	server_cpu=PT10H34M28.19999998S	client_received=10000/10000	server_resps=10000	codes={200=120, 500=9880}
+slowdown_and_error_thresholds[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=94.2%	client_mean=PT2M19.506447524S	server_cpu=PT6H33M46.759998892S	client_received=10000/10000	server_resps=10000	codes={200=9423, 500=577}
+    slowdown_and_error_thresholds[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=91.8%	client_mean=PT2M10.117350732S	server_cpu=PT9H7M52.566666203S	client_received=10000/10000	server_resps=10000	codes={200=9184, 500=816}
```

Comment:
Guys this is about as dramatic as possible. 1.2% user-perceived success rate turned into 91.8%.

https://github.com/palantir/dialogue/blob/dfox/bring-back-windowed-limit/simulation/src/test/resources/report.md#slowdown_and_error_thresholdsconcurrency_limiter_round_robin